cmd/pocket_core/main.go

```diff
@@ -20,6 +23,8 @@ func main() {
 "startClient" Starts the client with the given initial configuration.
  */
 func startClient(){
+	logs.LogConstructorAndLog("test.log","pocket-core/cmd/pocket_core/main.go",
+		"27", "Testing testing 123", logrus.DebugLevel, &logmatic.JSONFormatter{})
```

Comment:
A higher level interface shouldn't concern itself with how the logger works, there should be an abstraction so that the logger package implementation could switch without having to change the interface.

Reply:
Check it out now, should fix all